packages/schema/src/doc.ts

```diff
@@ -4,6 +4,8 @@ import { ISignal, Signal } from '@lumino/signaling';
 import * as Y from 'yjs';
 
 import { IJCadObject, IJCadOptions } from './_interface/jcad';
+export { SCHEMA_VERSION } from './_interface/version';
```

Comment:
it can be exported directly in `index.ts`?

Reply:
Sounds good!